### PR TITLE
Firebase and react setup

### PR DESCRIPTION
**Pull request by LightFelicis**
To deploy this site on our firebase server, you will need to use
`firebase login`
first

### CODE REVIEW

**Comment by LightFelicis** [comment]
@googlebot I signed it!

File: README.old.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# Capstone project
```
Comment (leon-kastler-google) [inline README.old.md:1]: We can remove this file.

File: package.json [hunk 2]
```diff
@@ -0,0 +1,34 @@
+{
+  "name": "step236-2020",
```
Comment (leon-kastler-google) [inline package.json:2]: we can have a more expressive name here.

File: public/manifest.json [hunk 3]
```diff
@@ -0,0 +1,25 @@
+{
+  "short_name": "React App",
+  "name": "Create React App Sample",
```
Comment (leon-kastler-google) [inline public/manifest.json:3]: do we need more expressive names here?

**Review by leon-kastler-google — APPROVED** [review]

File: public/manifest.json (hunk 3 above)
Comment (LightFelicis) [reply public/manifest.json:3]: Replaced with scriba. It's the name of app, so maybe title will suit better than generic "React app".

File: package.json (hunk 2 above)
Comment (LightFelicis) [reply package.json:2]: Replaced with scriba. Do we need more descriptive, or only title is ok?

File: README.old.md (hunk 1 above)
Comment (LightFelicis) [reply README.old.md:1]: Done.

**Review by xmxwx — APPROVED** [review]

**Pull request merged**